OpenRA.Game/Actor.cs

```diff
@@ -295,6 +295,11 @@ public bool IsDisabled()
 			return false;
 		}
 
+		public bool IsInvisibleTo(Player player)
```

Comment:
I can't use `CanViewActor` as it includes fog visibility so this isn't duplication.